README.md

```diff
@@ -115,12 +115,11 @@ The ROS 1 node will start printing the published messages to the console.
 ---
 
 Now we start the ROS 2 listener.
-It is important to use the listener from the `ros1_bridge` package since it uses different QoS parameters than the "normal" `listener`.
 
 ```
 # Shell D:
 . <install-space-with-ros2>/setup.bash
-ros1_bridge_listener
+listener
```

Comment:
ok sounds good, this executable is now in the `rclcpp_tutorials` package (for the readme)